TrajectoryPredictorConfig:
    # Kalman Filters
    acceleration_variance: 0.5
    measurement_variance: 0.5
    initial_uncertainty_guess: 10

    # Trajectory search
    arm_length: 0.4
    robot_base:
      - -0.1101
      - 0.000635
      - 0.1
    sufficient_convergence: 0.05
    search_delta: 0.01

    min_time_delta: 1
    max_time_delta: 10
    
